README.md

```diff
@@ -34,6 +34,7 @@ markdownlint --help
     -f, --fix                                   fix basic errors (does not work with STDIN)
     -i, --ignore [file|directory|glob]          file(s) to ignore/exclude
     -o, --output [outputFile]                   write issues to file (no console)
+    -j, --json                                  write issues in json format
```

Comment:
Please sort alphabetically.

Reply:
sure. I had missed the fact they were sorted.

Reply:
I think I'm looking at the latest changes, but it's still after -o instead of before it?

Reply:
Oh, it's sorted in the code, but not the README here.